src/Renderer/ThreeExtended/B3dmLoader.js

```diff
@@ -62,24 +76,23 @@ B3dmLoader.prototype.parse = function parse(buffer) {
         b3dmHeader.BTBinaryLength = view.getUint32(byteOffset, true);
         byteOffset += Uint32Array.BYTES_PER_ELEMENT;
 
-        // To test with a binary which possess a batch table
-        /* if(result.header.batchTableJSONByteLength > 0) {
-            var batchedTableString = this.decodeFromCharCode(array.subarray(result.byteOffset, result.byteOffset + result.header.batchTableJSONByteLength));
-            result.body.batchTableJSON = JSON.parse(batchedTableString);
-            result.byteOffset += result.header.batchTableJSONByteLength;
-        }
-        if(result.header.batchTableBinaryByteLength > 0) {
-            var batchTableBinary = new Uint8Array(buffer, result.byteOffset, result.batchTableBinaryByteLength);
-            batchTableBinary = new Uint8Array(batchTableBinary);
-            result.byteOffset += result.header.batchTableBinaryByteLength;
-        }*/
-
         // TODO: missing feature and batch table
         return new Promise((resolve/* , reject*/) => {
             const onload = (gltf) => {
                 for (const scene of gltf.scenes) {
                     scene.traverse(filterUnsupportedSemantics);
                 }
+                // Rotation managed
+                if (gltfUpAxis === undefined || gltfUpAxis === 'Y') {
+                    gltf.scene.applyMatrix(matrix4_X);
```

Comment:
The naming of the matrix is a bit strange.

Maybe `matrix4_X` could be renamed `matrixChangeUpVectorZtoY`  - it's a bit long, but it's clearer :)